src/app/shared/components/timer/timer.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnDestroy } from '@angular/core';
+import { interval, Subscription } from 'rxjs';
+
+const rxTimer = interval(10);
```

Comment:
no need to keep this here

Reply:
I move interval out of class because don't want to recall this code every time when component create. Logical this variable should create only one time and store while it required and after some time remove. Maybe this functional required small amount of resource so will move to class.

Reply:
you don't need `this.clock = null;` when using `stop$: Subject<void> = new Subject();` so you can initialize `clock` only once